ui/src/app/about/components/about-more/about-details.component.html

```diff
@@ -0,0 +1,261 @@
+    </div>
+
+
+    <div class="box">
+      <span class="fa fa-code-fork"></span>
+      <h2>Version Information</h2>
+      <div id="versionInformationTable">
+        <div class="line">
+          <span>Implementation Version:</span>
+          <strong id="implementationVersion">
+            {{ dataflowVersionInfo.versionInfo.implementation.version }} ({{ dataflowVersionInfo.versionInfo.implementation.name }})
+          </strong>
+        </div>
+        <div class="line">
+          <span>Core:</span>
+          <strong id="coreVersion">
+            {{ dataflowVersionInfo.versionInfo.core.version }} ({{ dataflowVersionInfo.versionInfo.core.name }})
+          </strong>
+        </div>
+        <div class="line">
+          <span>Dashboard:</span>
+          <strong id="dashboardVersion">
+            {{ dataflowVersionInfo.versionInfo.dashboard.version }} ({{ dataflowVersionInfo.versionInfo.dashboard.name }})
+          </strong>
+        </div>
+        <div class="line">
+          <span>Shell:</span>
+          <strong id="shellVersion">
+            {{ dataflowVersionInfo.versionInfo.shell.version }} ({{ dataflowVersionInfo.versionInfo.shell.name }})
+          </strong>
+        </div>
+        <div id="shellChecksumSha1" class="line" *ngIf="dataflowVersionInfo.versionInfo.shell.checksumSha1">
+          <span>Shell Checksum Sha1:</span>
+          <strong>{{ dataflowVersionInfo.versionInfo.shell.checksumSha1 }}</strong>
+        </div>
+        <div id="shellChecksumSha256" class="line" *ngIf="dataflowVersionInfo.versionInfo.shell.checksumSha256">
+          <span>Shell Checksum Sha256:</span> <strong>{{ dataflowVersionInfo.versionInfo.shell.checksumSha256 }}</strong>
+        </div>
+      </div>
+    </div>
+
+    <div class="box">
+      <span class="fa fa-cube"></span>
+      <h2 *ngIf="dataflowVersionInfo.featureInfo.skipperEnabled">Runtime Environment - Skipper Deployer</h2>
+      <h2 *ngIf="!dataflowVersionInfo.featureInfo.skipperEnabled">Runtime Environment - App Deployer</h2>
+      <div id="appDeployerTable">
+        <div class="line">
+          <span>Implementation Version:</span>
+          <strong>{{ dataflowVersionInfo.runtimeEnvironment.appDeployer.deployerImplementationVersion }}</strong>
+        </div>
+        <div class="line">
+          <span>Name:</span>
+          <strong>{{ dataflowVersionInfo.runtimeEnvironment.appDeployer.deployerName }}</strong>
+        </div>
+        <div class="line">
+          <span>Spi Version:</span>
+          <strong>{{ dataflowVersionInfo.runtimeEnvironment.appDeployer.deployerSpiVersion }}</strong>
+        </div>
+        <div class="line">
+          <span>Java Version:</span>
+          <strong>{{ dataflowVersionInfo.runtimeEnvironment.appDeployer.javaVersion }}</strong>
+        </div>
+        <div class="line">
+          <span>Platform Api Version:</span>
+          <strong>{{ dataflowVersionInfo.runtimeEnvironment.appDeployer.platformApiVersion }}</strong>
```

Comment:
Since we don't use a table anymore, we should visually point out that no value is available. So may a `N/A`?